sdk/core/Azure.Core/tests/public/Azure.Core.Tests.Public.csproj

```diff
@@ -20,12 +20,19 @@
   <!-- Shared source from Azure.Core -->
   <ItemGroup>
```

Comment:
We should validate that tests failing in this project will fail the CI - I'm not sure I completed that work for DynamicData.